src/main/java/de/tum/in/www1/artemis/repository/ResultRepository.java

```diff
@@ -650,6 +650,10 @@ default Result findFirstWithFeedbacksByParticipationIdOrderByCompletionDateDescE
                 .orElseThrow(() -> new EntityNotFoundException("Result by participationId", participationId));
     }
 
+    default Result findWithEagerSubmissionAndFeedbackAndAssessorByIdElseThrow(long resultId) {
+        return findWithEagerSubmissionAndFeedbackAndAssessorById(resultId).orElseThrow(() -> new EntityNotFoundException("Result", resultId));
+    }
```

Comment:
Same as above

Reply:
I just noticed that this is the case with a lot more methods. I don't know if it's a convention, so I cannot really say if this should be changed.

Reply:
This repository has the default methods grouped next to each other. I don't think there is a convention here, but I just kept it like it was.